<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKToolBoxUiCommunicationMenu</class>
 <widget class="QWidget" name="SAKToolBoxUiCommunicationMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="SAKAnalyzerToolUi" name="widgetTxAnanyzer" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="SAKAnalyzerToolUi" name="widgetRxAnanlyzer" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="SAKMaskerToolUi" name="widgetTxMasker" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="SAKMaskerToolUi" name="widgetRxMasker" native="true"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="SAKStorerToolUi" name="widgetStorer" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAKAnalyzerToolUi</class>
   <extends>QWidget</extends>
   <header location="global">SAKAnalyzerToolUi.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SAKMaskerToolUi</class>
   <extends>QWidget</extends>
   <header location="global">SAKMaskerToolUi.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SAKStorerToolUi</class>
   <extends>QWidget</extends>
   <header location="global">SAKStorerToolUi.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
